src/implementations/DopplerLegacyClientImpl.ts

```diff
@@ -34,6 +34,25 @@ export class DopplerLegacyClientImpl implements DopplerLegacyClient {
       value: { items: response.data.images.map(parseImageItem) },
     };
   }
+
+  async uploadImage(file: File): Promise<Result> {
+    // TODO: deal with server errors for example:
+    //   {"success":false,"error":"Tamaño inválido","maxSize":3145728}
+    // TODO: do client side validations like this:
+    //   https://github.com/MakingSense/MSEditor/blob/v1.4.0/app/controllers/mseditorImageGalleryCtrl.js
+    //   Lines #L95-L105
```

Comment:
@jcamposmk 

> Con respecto a las validaciones como la estas manejando? solo con el atributo accept?

As you can see here, we are not dealing with client-side validations or server errors yet.

> At the moment, it is not dealing with errors, so, it simply show the error in the console: 
> 
> https://github.com/FromDoppler/doppler-editors-webapp/assets/1157864/eb63c873-9627-4cdd-ab01-23c59747d7a9
 
See related ticket [DOP-1362](https://makingsense.atlassian.net/browse/DOP-1362) and [chat](https://makingsense.slack.com/archives/C01JUFJSX4N/p1685643429992649?thread_ts=1685643292.135899&cid=C01JUFJSX4N).

At the moment we are simply ignoring the error and, of course, the image is not added.